configure.go

```diff
@@ -0,0 +1,80 @@
+package discovery
+
+import (
+	"errors"
+	"fmt"
+	"net/url"
+	"os"
+	"strings"
+
+	kv "gopkg.in/clever/kayvee-go.v2"
+)
+
+// m is a convenience type for using kv.
+type m map[string]interface{}
+
+const (
+	templateVar = "SERVICE_%s_%s_%%s"
+)
+
+func getVar(envVar string) (string, error) {
+	envVar = strings.ToUpper(envVar)
+	val := os.Getenv(envVar)
+	if val == "" {
+		return "", errors.New(kv.FormatLog("discovery-go", kv.Error, "missing env var", m{
+			"var": envVar,
+		}))
+	}
+	return val, nil
+}
+
+// URL finds the specified URL for a service based off of the service's name and which
+// interface you are accessing. Values are found in environment variables fitting the scheme:
+// SERVICE_{SERVICE NAME}_{INTERFACE NAME}_{PROTO,HOST,PORT}.
+func URL(service, name string) (string, error) {
```

Comment:
Better late than never!

Adding INTERFACE_NAME accounts for a handful of services that expose multiple ports with different protocols. [Influxdb-service](https://github.com/Clever/influxdb-service/blob/master/launch/influxdb-service.yml) is one example.
There aren't a ton of services that have multiple interfaces but this gives us a lot of future extendability.